ml-conversational-analytic-tool/githubDataExtraction.py

```diff
@@ -160,18 +154,26 @@ def getPullFeatures(self, pull):
 if __name__ == "__main__":
     parser = argparse.ArgumentParser(description='Create csv for all pulls in repo')
     parser.add_argument('organization', help='Organization the repo belongs to.')
-    parser.add_argument('reponame', help='Name of repo')
-    parser.add_argument('-reactions', action='store_true', default=False, help='Flag to extract reactions')
-    parser.add_argument('--filename', help='Name of file')
-    # parser.add_argument('accesstoken', help='Github access token')
+    parser.add_argument('--repo', help='Name of repo.')
+    parser.add_argument('--reactions', action='store_true', default=False, help='Flag to extract reactions')
+    parser.add_argument('-n', "--name", help='Output file name. If not specified, the name is constructed like this: '
+            '<organization>_<reponame>.csv')
 
     args = parser.parse_args()
     ACCESS_TOKEN = os.environ["GITACCESS"]  # Access Github token from environment for security purposes
     extractor = GithubDataExtractor(ACCESS_TOKEN)  # Create object
-    extractor.openRepo(args.organization, args.reponame)  # Open repo
 
-    # Extract all pulls and export them to .csv
-    if args.filename:
-        extractor.getAllPulls(args.filename, args.reactions)
+    if args.repo is None:
+        # Extract data for all repositories in organization
+        repos = extractor.g_ses.get_organization(args.organization).get_repos()
+        for repo in repos:
+            extractor.openRepo(args.organization, repo.name)
+            df = extractor.getAllPulls(args.reactions)
+            file_name = utils.construct_file_name(args.name, args.organization, repo.name)
```

Comment:
Wouldn't this create the same file name for every repo in the organization? Take a look at [Diana's comment](https://github.com/vmware-labs/ml-conversational-analytic-tool/pull/23#discussion_r787825723), we should make sure that `args.name` is used only when args.repo is not None

Reply:
The `args.name` should be `None` in this case, (as it is relevant just for single repo) 
To be more obvious - `file_name = utils.construct_file_name(None, args.organization, args.repo)` ?

Reply:
It shouldn't create the same file because it should use args.organization and repo.name to create the file name since args.name is None

Reply:
Doesn't it just assume arg.name is None? There is nothing that prevents the user from not passing args.repo but pass in args.name.

Reply:
Yes, there is nothing to prevent you. If you provide the first argument - this will be the name of the output file. (the other two arguments will be ignored)